lib/infra/infra-stack.ts

```diff
@@ -375,6 +377,33 @@ export class InfraStack extends Stack {
   private static getCfnInitElement(scope: Stack, logGroup: LogGroup, props: infraProps, nodeType?: string): InitElement[] {
     const configFileDir = join(__dirname, '../opensearch-config');
     let opensearchConfig: string;
+    const procstatConfig: ProcstatMetricDefinition[] = [{
+      pattern: '-Dopensearch',
+      measurement: [
+        'cpu_usage',
+        'cpu_time_system',
+        'cpu_time_user',
+        'read_bytes',
+        'write_bytes',
+        'pid_count',
+      ],
+      metrics_collection_interval: 10,
+    },
+    ];
+    if (props.dashboardsUrl !== 'undefined') {
+      procstatConfig.push({
+        pattern: 'opensearch-dashboards',
+        measurement: [
+          'cpu_usage',
+          'cpu_time_system',
+          'cpu_time_user',
+          'read_bytes',
+          'write_bytes',
+          'pid_count',
```

Comment:
Hi @rishabh6788 ,

Not sure if we need all of these metrics. Just pid_count will suffice right?

Reply:
Fixed it. Will only give pid_count metric now. Thanks!